ui/src/components/about/VisionMissionSection.js

```diff
@@ -0,0 +1,73 @@
+import {
+  Box,
+  Container,
+  VStack,
+  Stack,
+  Text,
+  Heading,
+} from "@chakra-ui/layout";
+
+const VisionMissionCard = ({ title, message }) => {
+  return (
+    <Box
+      w={["20em", "30em"]}
```

Comment:
element padding ought to be fixed lol, the width also looks real janky with a bigger container and it looks really spaced far out